www/front_src/src/Resources/index.tsx

```diff
@@ -398,13 +382,15 @@ const Resources = (): JSX.Element => {
           onStatesChange={changeStates}
           selectedStatuses={statuses}
           onStatusesChange={changeStatuses}
-          onSearchRequest={doSearch}
+          onSearchRequest={requestSearch}
+          onSearchPrepare={prepareSearch}
```

Comment:
this prop seems not managed by filter component.
Did you forget to commit `Filter/index.tsx file` ?

Reply:
It's defined here: https://github.com/centreon/centreon/blob/05a87ebae823f144f391c86b9ebaeb63c7bd1aca/www/front_src/src/Resources/Filter/index.tsx#L114

Reply:
Indeed my bad